qiling/os/utils.py

```diff
@@ -371,7 +371,10 @@ def printf(self, address, fmt, params_addr, name, wstring=False):
                 params.append(
```

Comment:
This is not valid on x64 for normal arguments, only when an argument list is passed in!